packages/dashboard/src/app/views/myStories/karma/myStoriesList.karma.js

```diff
@@ -0,0 +1,613 @@
+  });
+
+  async function getStoriesState() {
+    const {
+      state: { stories },
+    } = await fixture.renderHook(() => useApi());
+
+    return stories;
+  }
+
+  function isElementVisible(element) {
+    return Boolean(
+      element.offsetWidth ||
+        element.offsetHeight ||
+        element.getClientRects().length
+    );
+  }
+
+  const clickListView = async () => {
+    expect(listViewButton).toBeTruthy();
+
+    await fixture.events.click(listViewButton);
+  };
+
+  const enterListView = async () => {
+    expect(listViewButton).toBeTruthy();
+    await fixture.events.focus(listViewButton);
+    await fixture.events.keyboard.press('Enter');
+  };
+
+  describe('Creator can see stories in list view', () => {
+    it('should switch to List View', async () => {
+      await clickListView();
+
+      const listViewTable = fixture.screen.getByTestId('story-list-view');
+
+      await expectAsync(listViewTable).toHaveNoViolations();
+
+      expect(listViewTable).toBeTruthy();
+    });
+
+    it('should switch to List View and back to Grid View', async () => {
+      await clickListView();
+
+      const listViewTable = fixture.screen.getByTestId('story-list-view');
+
+      expect(listViewTable).toBeTruthy();
+      const gridViewButton = fixture.screen.getByLabelText(
+        new RegExp(`^${VIEW_STYLE_LABELS[VIEW_STYLE.LIST]}$`)
+      );
+
+      await fixture.events.click(gridViewButton);
+
+      const storyElements = fixture.screen.getAllByTestId(/^story-grid-item/);
+
+      const { storiesOrderById } = await getStoriesState();
+
+      expect(storyElements.length).toEqual(storiesOrderById.length);
+    });
+
+    it('should Rename a story', async () => {
+      const { stories, storiesOrderById } = await getStoriesState();
+
+      const storiesSortedByModified = storiesOrderById.map((id) => stories[id]);
+
+      await clickListView();
+
+      // drop the header row using slice
+      const rows = fixture.screen.getAllByRole('row').slice(1);
+
+      const utils = within(rows[1]);
+
+      const titleCell = utils.getByRole('heading', {
+        name: storiesSortedByModified[1].title,
+      });
+
+      await fixture.events.hover(titleCell);
+
+      const moreOptionsButton = utils.getByRole('button', {
+        name: /^Context menu for/,
+      });
+
+      await fixture.events.click(moreOptionsButton);
+
+      const rename = utils.getByText(/^Rename/);
+
+      await fixture.events.click(rename);
+
+      const input = await utils.getByRole('textbox');
+
+      const inputLength = input.value.length;
+
+      for (let iter = 0; iter < inputLength; iter++) {
+        // disable eslint to prevet overlapping .act calls
+        // eslint-disable-next-line no-await-in-loop
+        await fixture.events.keyboard.press('Backspace');
+      }
+
+      await fixture.events.keyboard.type('A New Title');
+
+      await fixture.events.keyboard.press('Enter');
+
+      expect(utils.getByText(/A New Title/)).toBeTruthy();
```

Comment:
Should this be more specific?

```suggestion
      expect(utils.getByText(/^A New Title$/)).toBeTruthy();
```